jetkit/api/auth/__init__.py

```diff
@@ -66,6 +69,24 @@ def login(email: str, password: str):
             abort(401, message="Wrong user name or password")
         return auth_response_for_user(user)
 
+    @blp.route("sign-up", methods=["POST"])
+    @blp.response(user_schema)
+    @blp.arguments(LoginRequest, as_kwargs=True)
+    def sign_up(email: str, password: str):
+        """Sign up with email and password. Possibly add other fields later."""
+        cleaned_email = email.strip().lower()
+        existing_user: AuthModel = auth_model.query.filter_by(email=cleaned_email).one_or_none()
+        if existing_user:
+            abort(400, message="There's already a registered user with this email")
+        values = dict(email=email, _password=generate_password_hash(password), _user_type=CoreUserType.normal.value)
```

Comment:
IDK, is there currently in jetkit a better way to insert a user?

Reply:
There is really no reason to use upsert here at all.
what's wrong with:
```python
user = User(email=email, password=password)
session = auth_model.query.session
session.add(user)
session.commit()
```